lib/console.js

```diff
@@ -83,7 +83,10 @@ function createWriteErrorHandler(stream) {
       // an `error` event. Adding a `once` listener will keep that error
       // from becoming an uncaught exception, but since the handler is
       // removed after the event, non-console.* writes won’t be affected.
-      stream.once('error', noop);
+      // we are only adding noop if there is no one else listening for 'error'
```

Comment:
if this can happen only with `writev()`, its worth mentioning specifically in the comment